apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -0,0 +1,144 @@
+export function InstructorCourseAdminSettings({
+  resLocals,
+  needToSync,
+  noInfo,
+}: {
+  resLocals: Record<string, any>;
+  needToSync: boolean;
+  noInfo: boolean;
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", { ...resLocals })}
+        <style>
+          .popover {
+            max-width: 50%;
+          }
+        </style>
+      </head>
+      <body>
+        <script>
+          $(function () {
+            $('[data-toggle="popover"]').popover({
+              sanitize: false,
+            });
+          });
+        </script>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", { ...resLocals })}
+        <main id="content" class="container-fluid">
+          ${renderEjs(__filename, "<%- include('../partials/courseSyncErrorsAndWarnings'); %>", {
+            ...resLocals,
+          })}
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex">
+              Course ${resLocals.course.short_name}
+            </div>
+            <table class="table table-sm table-hover two-column-description">
+              <tbody>
+                <tr>
+                  <th>Short Name</th>
+                  <td>${resLocals.course.short_name}</td>
+                </tr>
+                <tr>
+                  <th>Title</th>
+                  <td>${resLocals.course.title}</td>
+                </tr>
+                <tr>
+                  <th>Institution</th>
+                  <td>${resLocals.institution.short_name} (${resLocals.institution.long_name})</td>
+                </tr>
+                <tr>
+                  <th>Timezone</th>
+                  <td>${resLocals.course.display_timezone}</td>
+                </tr>
+                <tr>
+                  <th>Path</th>
+                  <td>${resLocals.course.path}</td>
+                </tr>
+                <tr>
+                  <th>Repository</th>
+                  <td>${resLocals.course.repository}</td>
+                </tr>
+                <tr>
+                  <th>Configuration</th>
+                  <td>
+                    ${needToSync
+                      ? resLocals.authz_data.has_course_permission_edit &&
+                        !resLocals.course.example_course
+                        ? html`
+                            <span class="text-danger">
+                              You must
+                              <a href="<%= urlPrefix %>/<%= navPage %>/syncs">sync your course</a>
+                              before viewing or editing its configuration
+                            </span>
+                          `
+                        : html`
+                            <span class="text-danger">
+                              A course editor must sync this course before anyone can view or edit
+                              its configuration</span
+                            >
```

Comment:
Nit:

```suggestion
                              its configuration
                            </span>
```

Reply:
Updated locally in [91314d4](https://github.com/PrairieLearn/PrairieLearn/pull/9406/commits/91314d4c3e75e28d2bda3cfe367f712392d9d690).